Sources/mpc-core-kit-swift/Interface.swift

```diff
@@ -176,3 +176,94 @@ public struct enableMFARecoveryFactor {
         self.additionalMetadata = additionalMetadata
     }
 }
+
```

Comment:
Many of the structs declared here will be in the next version of customauth.

I'd suggest  we wait with this PR for that and then update the dependency instead and use it from there.